### PR TITLE
bio

### PR DESCRIPTION
**Pull request by hnbarr**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [ ] Effective use of box model: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 15 points

**Checkpoint 2**
- [ ] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [ ] Effective use of CSS Grid: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Site is hosted on github: 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 15 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 15 points

### CODE REVIEW

File: 02week/bio/css/style.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+body {
+    background-image: url(../edd.jpg);
```
Comment (DinhH) [inline 02week/bio/css/style.css:2]: it's conventional to create an img/image folder to store all of the images you would use for the project.

File: 02week/bio/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <p>Hi, My name is Heather! I love to have fun and to create unique webpages!
+        I currently use HTML with CSS <strong>(css is my fav),</strong> which is what this page is made of! And I will be learning javascript in a few weeks!
+        Check out some of my work at the tab above. If you like what you see, I'd love to collaborate or design something great for you!
+        </p>
+    </main>
+ <!-- how my viewers can reach me! -->
+<footer>
+        <a href="mailto:hnbarr94@gmail.com"><i class="far fa-envelope-open"></i></a>
+        <a href="https://www.linkedin.com/in/heather-barr-5b71a6170/"><i class="fab fa-linkedin-in"></i></a>
+        <a href="https://github.com/hnbarr"><i class="fab fa-github"></i></a>
```
Comment (DinhH) [inline 02week/bio/index.html:35]: Love the use of semantic tags

**Comment by DinhH** [comment]
great work on the project!

File: 02week/bio/css/style.css (hunk 1 above)
Comment (hnbarr) [reply 02week/bio/css/style.css:2]: Okay, I will definitely take that approach, moving forward! Thank you!

File: 02week/bio/index.html (hunk 2 above)
Comment (hnbarr) [reply 02week/bio/index.html:35]: Thank you!

**Pull request merged**